exercises/square-root/introduction.md

```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+We are launching a deep space exploration rocket and we need a way to make sure the navigation system stays on target.
+
+The first step in our coordinate calculation involves taking a number as input and finding another number which multiplied by itself will equal our input number, or in other words, finding the square root of our input number.
```

Comment:
This sentence is hard for me to follow, even as a native English speaker. It might be worth trying to break this into smaller pieces.